spec/static/spec/v0.9/openapitransactional.yaml

```diff
@@ -899,7 +899,8 @@ paths:
           example: yLLiDqX2OL7mcIMTjob60A==
       responses:
         "200":
-          description: Asset file uploaded part successfully
+          description: Asset file uploaded part successfully (No Content).
```

Comment:
Could it also be "asset file part uploaded" (order of "uploaded" and "part" changed)?